Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/SMGs/p90.yml

```
@@ -0,0 +1,157 @@
+      FullAuto:
+        maxScatterModifier: 3
+        useBurstScatterMult: true
+        unwieldedScatterMultiplier: 2
+        shotsToMaxScatter: 4
+  - type: RMCWeaponAccuracy
+    accuracyMultiplier: 1
+    accuracyMultiplierUnwielded: 0.9
+  - type: MagazineAmmoProvider
+    autoEject: false
+    soundAutoEject: null
+  - type: WieldableSpeedModifiers
+    base: 0.725
+    light: 1
+    medium: 1
+    heavy: 1
+  - type: ItemSlots
+    slots:
+      gun_magazine:
+        name: Magazine
+        startingItem: CMMagazineSMGP90
+        insertSound: /Audio/_RMC14/Weapons/Guns/Reload/smg_reload.ogg
+        ejectSound: /Audio/_RMC14/Weapons/Guns/Reload/smg_unload.ogg
+        priority: 2
+        whitelist:
+          tags:
+          - CMMagazineSMGP90
+  - type: AttachableHolder
+    randomAttachmentChance: 0.1
+    slots:
+      rmc-aslot-barrel:
+        whitelist:
+          tags:
+          - RMCAttachmentBarrelCharger
+          - RMCAttachmentExtendedBarrel
+          - RMCAttachmentRecoilCompensator
+          - RMCAttachmentSuppressor
+          - RMCM5Bayonet
+      rmc-aslot-rail:
+        whitelist:
+          tags:
+          - RMCAttachmentMagneticHarness
+          - RMCAttachmentS5RedDotSight
+          - RMCAttachmentS6ReflexSight
+          - RMCAttachmentS42xTelescopicMiniscope
+  - type: AttachableHolderVisuals
+    offsets:
+      rmc-aslot-barrel: 0.84, -0.02
+      rmc-aslot-rail: 0.2, 0.219
+  - type: GunDamageModifier
+    multiplier: 1.2
+
+- type: entity
+  parent: BaseMagazineRifle
+  id: CMMagazineSMGP90
+  name: "FN P90 magazine (5.7×28mm)"
+  description: "A 5.7×28mm magazine for the FN P90."
+  components:
+  - type: Tag
+    tags:
+    - CMMagazineSmg
+    - CMMagazineSMGP90
+  - type: BallisticAmmoProvider
+    mayTransfer: True
+    whitelist:
+      tags:
+      - CMCartridge57×28mm
+    proto: CMCartridge57×28mm
+    capacity: 50
+  - type: Sprite
+    sprite: _RMC14/Objects/Weapons/Guns/Ammunition/Magazines/p90.rsi
+    layers:
+    - state: base
+      map: ["enum.GunVisualLayers.Base"]
+    - state: mag-1
+      map: ["enum.GunVisualLayers.Mag"]
+  - type: MagazineVisuals
+    magState: mag
+    steps: 2
+    zeroVisible: false
+    zeroOnlyOnEmpty: true
+  - type: Appearance
+
+- type: entity
+  parent: CMCartridge10x20mm
+  id: CMCartridge57×28mm
+  name: cartridge (57×28mm SMG)
+  description: A 5.7×28mm magazine for the FN P90.
+  components:
+  - type: Tag
+    tags:
+    - Cartridge
+    - CMCartridge57×28mm
+  - type: CartridgeAmmo
+    proto: CMBullet57×28mm
+
+- type: entity
+  parent: Bullet10x20mm
+  id: CMBullet57×28mm
+  components:
+  - type: Projectile
+    damage:
+      types:
+        Piercing: 26
+  - type: RMCProjectileDamageFalloff
+    thresholds:
+    - range: 22
+      falloff: 9999
+      ignoreModifiers: true
+    - range: 8
+      falloff: 5
+  - type: CMArmorPiercing
+    amount: 10
+
+- type: Tag
+  id: CMMagazineSMGP90
+
+- type: Tag
+  id: CMCartridge57×28mm
```

```suggestion
  id: RMCCartridge57x28mm
```
dont use `×`, use `x` instead.